setup_workspace/setup_workspace.sh

```diff
@@ -53,5 +56,39 @@ else
   echo 'No changes were made to client'
 fi
 
+if [ "$1" = "python" ]; then
+  # Fix for compatibility with Python 3.7
+  LC_ALL=C find docs rapid7vmconsole samples setup_workspace test -type f -exec sed -i '' 's/async_=params.get/async_=params.get/g' {} +
+  LC_ALL=C find docs rapid7vmconsole samples setup_workspace test -type f -exec sed -i '' 's/async_=None/async_=None/g' {} +
+  LC_ALL=C find docs rapid7vmconsole samples setup_workspace test -type f -exec sed -i '' 's/if not async_____/if not async______/g' {} +
+
+  # Cleanup API documentation strings
+  LC_ALL=C find . -type f -exec perl -0777 -i -pe 's/InsightVM API[\s]*# Overview.*/Python InsightVM API Client/' {} +
+
+  # Cleanup local console details
+  LC_ALL=C find . -name "*.py" -type f -exec sed -i '' "s/$2:3780/localhost:3780/g" {} +
+fi
+
+if [ "$1" = "ruby" ]; then
+  # Cleanup API documentation strings
+  LC_ALL=C find . -type f -exec perl -0777 -i -pe 's/#InsightVM API\n\n## Overview.*/Ruby InsightVM API Client/' {} +
+
+  # Cleanup local console details
+  LC_ALL=C find . -name "*.rb" -type f -exec sed -i '' "s/$2:3780/localhost:3780/g" {} +
+
+  # Exclude the setup_workspace directory from the gemspec
+  LC_ALL=C perl -0777 -i -pe 's/find \*/find docs lib spec/' rapid7_vm_console.gemspec
+fi
+
+
+if [ "$1" = "go" ]; then
+  # Cleanup API documentation strings
+  LC_ALL=C find . -type f -exec perl -0777 -i -pe 's/\* InsightVM API(\n\s\*){2} # Overview.*/* Go InsightVM Client/' {} +
```

Comment:
For consistency sake, did you mean to have a * before "Go InsightVM Client?

Reply:
I should have escaped it for consistency, but it does work as expected

Reply:
Fixed this so it's consistent